cms/static/js/views/course_rerun.js

```diff
@@ -44,6 +44,7 @@ define(['domReady', 'jquery', 'underscore', 'js/views/utils/create_course_utils'
             CreateCourseUtils.create(course_info, function(errorMessage) {
                 $('.wrapper-error').addClass('is-shown').removeClass('is-hidden');
                 $('#course_rerun_error').html('<p>' + errorMessage + '</p>');
+                // xss-lint: disable=javascript-jquery-html
```

Comment:
It would be better to not disable and to simply use `.text()` instead of `.html()`.